### PR TITLE
Remove outdated examples

### PR DESCRIPTION
**Pull request by iNikem**
This PR removes a LOT of obsolete examples from this repository:

- Everything that mentions OpenTracing
- Everything that mentions Jaeger
- SignalFx examples for languages where we already have OpenTelemetry distributions

Remaining examples are these
Screenshot 2021-12-15 at 12 09 02

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,11 +98,7 @@ Their detailed documentation is available in their respective source locations:
 
 - [.NET](https://github.com/signalfx/signalfx-dotnet-tracing)
 - [Go](https://github.com/signalfx/signalfx-go-tracing)
-- [Java](https://github.com/signalfx/signalfx-java-tracing)
-- [Node.js](https://github.com/signalfx/signalfx-nodejs-tracing)
 - [PHP](https://github.com/signalfx/signalfx-php-tracing)
-- [Python](https://github.com/signalfx/signalfx-python-tracing)
-- [Ruby](https://github.com/signalfx/signalfx-ruby-tracing)
```
Comment (owais) [inline README.md:134]: We don't have Otel Ruby yet

**Review by owais — COMMENTED** [review]
Looks good but Ruby shouldn't be removed just yet as the otel distro is not ready

File: README.md [hunk 2]
```diff
@@ -114,11 +90,6 @@ instrument .NET libraries and applications.
 - [AWS App Mesh](./service-mesh/appmesh): An example E-Commerce application with
   deployment and configuration files for App Mesh on ECS.
 
-### AWS Lambda Functions
-
-- [AWS Lambda](./aws-lambda): Examples for instrumenting spans for AWS Lambda written in Java, Python, Node, Go
```
Comment (mateuszrzeszutek) [inline README.md:119]: Do we have any `splunk-otel-lambda` examples? @kubawach

File: README.md (hunk 1 above)
Comment (iNikem) [reply README.md:134]: There was not `signalfx-ruby-tracing` folder even before this PR :D

File: README.md (hunk 2 above)
Comment (kubawach) [reply README.md:119]: Yep, in the opentelemetry-tracing / opentelemetry-lambda. I took a look at @iNikem changes and I'm OK with them.

**Review by kubawach — APPROVED** [review]
Love this PR.

File: README.md (hunk 1 above)
Comment (iNikem) [reply README.md:134]: @owais will you approve now or should I do something about this? :)

**Review by owais — APPROVED** [review]

**Review by owais — APPROVED** [review]

**Pull request merged**